antora-ui-camel/src/partials/footer-scripts.hbs

```diff
@@ -1,10 +1,15 @@
 <script src="{{uiRootPath}}/js/site.js"></script>
 <script src="{{uiRootPath}}/js/vendor/highlight.js"></script>
-<script type="application/ld+json">
-    {
-    "@context": "http://schema.org/",
-    "@type": "Organization",
-    "url": "{{uiRootPath}}",
-    "logo": "{{uiRootPath}}/img/logo-d.svg"
-    }
+<script type='application/ld+json'>
+{
+  "@context": "http://www.schema.org",
```

Comment:
```suggestion
  "@context": "http://schema.org",
```

Looking [at the schema.org](https://schema.org/Organization) I think this needs to be without `www`